src/raft.rs

```diff
@@ -389,6 +389,20 @@ impl<T: Storage> Raft<T> {
         self.batch_append = batch_append;
     }
 
+    /// Uses a new quorum function.
+    pub fn set_quorum(&mut self, quorum_fn: fn(usize) -> usize) {
```

Comment:
Could you also add `get_quorum`? So we can use the field in raftstore directly for `check_conf_change`.